mediathread/main/tests/test_views.py

```diff
@@ -1133,8 +1134,10 @@ def test_email_invite_invalid_email(self):
             in response.cookies['messages'].value)
 
     def test_accept_invite_invalid_uuid(self):
-        # no uuid
-        url = reverse('course-invite-accept', kwargs={'uidb64': None})
+        # random new uuid
+        url = reverse(
```

Comment:
Why change the test here from None to random? Are the kwargs required in Django 2.2?

Reply:
Basically - yeah. Django 2.2 causes the request with `'uidb64': None` to fail with an exception. We could catch that exception if we want to test the `/accounts/invite/accept/None/` route... but that seemed sort of contrived, and what real situation would actually cause that request to happen?

So, instead I'm just testing with a random, invalid UUID here. I suppose we could also test that just getting the URL with a random non-uuid string fails as gracefully as possible, too.